setup.py

```diff
@@ -0,0 +1,18 @@
+from distutils.core import setup
+
+execfile('Tribler/Core/version.py')
```

Comment:
@whirm 
this answer kind of scared me away from doing that: http://stackoverflow.com/a/2073599
but since Tribler dependencies are installed manually anyway, it doesn't really matter. i'll change it.